sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -261,7 +261,7 @@
     }
 
     public static class Versions {
-        public static final String CURRENT_VERSION = "2018-12-31";
+        public static final String CURRENT_VERSION = "2020-07-15";
```

Comment:
"2018-12-31"; //  non-partitioned collection deprecation and migration of existing collections
"2020-07-15"; // Subpartitioned Collections

.NET didn't raise API version before.
Why is the API version raise needed?

Reply:
We have raised the API version for V3 SDK too. It is required since backend disallows any actions on subpartitioned containers below 2020-07-15